DataFormats/TrackReco/src/HitPattern.cc

```diff
@@ -32,6 +32,19 @@ HitPattern::HitPattern(const HitPattern& other)
   memcpy(this->hitPattern, other.hitPattern, sizeof(uint16_t) * HitPattern::ARRAY_LENGTH);
 }
 
+HitPattern::HitPattern(const Run3ScoutingHitPatternPOD& other)
+    : hitCount(other.hitCount),
+      beginTrackHits(other.beginTrackHits),
+      endTrackHits(other.endTrackHits),
+      beginInner(other.beginInner),
+      endInner(other.endInner),
+      beginOuter(other.beginOuter),
+      endOuter(other.endOuter) {
+  const static unsigned short max_vector_length_ =
```

Comment:
```suggestion
  const static unsigned short max_vector_length =
```
Would avoid the final `_` (usually used for class members). What's the advantage of using `static` here?

Reply:
Done, including removal of `static` (no advantage).